packages/flutter_tools/test/general.shard/runner/flutter_command_test.dart

```diff
@@ -1671,3 +1753,44 @@ class _TestRunCommandThatOnlyValidates extends RunCommand {
   @override
   bool get shouldRunPub => false;
 }
+
+class FakeFeature extends Feature {
+  const FakeFeature({required super.name, super.runtimeId, required this.enabled});
+
+  final bool enabled;
+}
+
+class FakeFeatureFlags implements FeatureFlags {
+  const FakeFeatureFlags({required this.allFeatures});
+
+  @override
```

Comment:
This can just be:

```dart
@override
Object? noSuchMethod(Invocation invocation) => super.noSuchMethod(invocation);
```

That effectively does the same thing without writing all of these out/having to change them over time.